manager/controllers/advisory_detail.go

```diff
@@ -142,57 +124,49 @@ func getAdvisoryFromDB(db *gorm.DB, advisoryName string) (*models.AdvisoryMetada
 		RebootRequired:   advisory.RebootRequired,
 		ReleaseVersions:  releaseVersions,
 	}
-	return &advisory, &ada, err
-}
-
-func getAdvisoryFromDBV2(db *gorm.DB, advisoryName string) (*AdvisoryDetailResponseV2, error) {
-	advisory, ada, err := getAdvisoryFromDB(db, advisoryName)
-	if err != nil {
-		return nil, err
-	}
 
 	pkgs, err := parsePackages(advisory.PackageData)
 	if err != nil {
 		return nil, errors.Wrap(err, "packages parsing error")
 	}
 
-	var resp = AdvisoryDetailResponseV2{Data: AdvisoryDetailItemV2{
-		AdvisoryDetailItem: AdvisoryDetailItem{ID: advisory.Name, Type: "advisory"},
-		Attributes: AdvisoryDetailAttributesV2{
-			AdvisoryDetailAttributes: *ada,
-			Packages:                 pkgs,
-		},
+	var resp = AdvisoryDetailResponse{Data: AdvisoryDetailItem{
+		ID:         advisory.Name,
+		Type:       "advisory",
+		Attributes: ada,
 	}}
+	resp.Data.Attributes.Packages = pkgs
+
 	return &resp, nil
 }
 
-func parsePackages(jsonb []byte) (packagesV2, error) {
+func parsePackages(jsonb []byte) (packages, error) {
 	if jsonb == nil {
-		return packagesV2{}, nil
+		return packages{}, nil
 	}
 
 	var err error
-	pkgsV2, err := parseJSONList(jsonb)
+	pkgs, err := parseJSONList(jsonb)
 	if err != nil {
 		// HACK!
 		// Until vmaas-sync syncs new data, `jsonb` has '{"<name>": "<evra>"}' format
-		// what we need for V2 api is ["<name>-<evra>", ...]
+		// what we need for V3 api is ["<name>-<evra>", ...]
 		// 1. try to unmarshal to packagesV1 struct
 		var tmpPkgV1 packagesV1
 		if v1err := json.Unmarshal(jsonb, &tmpPkgV1); v1err != nil {
-			// cannot unmarshal to neither V1 nor V2
+			// cannot unmarshal to neither V1 nor V3
 			return nil, err
 		}
-		// 2. create `packagesV2` from `packagesV1` data
+		// 2. create `packages` from `packagesV1` data
 		for k, v := range tmpPkgV1 {
-			// NOTE: V2 now shows the same data as V1 api until vmaas is synced
-			pkgsV2 = append(pkgsV2, fmt.Sprintf("%s-%s", k, v))
+			// NOTE: V3 now shows the same data as V1 api until vmaas is synced
+			pkgs = append(pkgs, fmt.Sprintf("%s-%s", k, v))
 		}
 	}
-	return pkgsV2, nil
+	return pkgs, nil
 }
```

Comment:
this function can be probably rewritten to load data directly to `packages` ([]string) instead of packagesV1 (map[string]string)